packages/driver/cypress/integration/e2e/multi-domain/multi_domain_uncaught_errors_spec.ts

```diff
@@ -155,6 +175,25 @@ describe('multi-domain - uncaught errors', { experimentalSessionSupport: true, e
         cy.get('.error-two').invoke('text').should('equal', 'promise rejection')
       })
     })
+
```

Comment:
should we also add a test for `spec` errors after the callback window is closed?

Reply:
I had thought about that but due to the potential timing issues I decided against it. Thoughts?

Reply:
I could write a test where the wait is outside of the `switchToDomain`. I would probably skip the test for now since it should fail but will currently pass.

```
    it.only('fails the current test/command if async errors are thrown from the test code in switchToDomain while the callback window is now closed', () => {
      cy.switchToDomain('foobar.com', () => {
        setTimeout(() => {
          throw new Error('setTimeout error')
        }, 50)
      })

      cy.wait(250)
    })
```

Reply:
I went ahead and added the tests for callback closed and added a skip to them. 1dd8269